tools/depth-quality/CMakeLists.txt

```diff
@@ -19,6 +19,13 @@ set( ELPP_FILES
     
 include(../../common/CMakeLists.txt)
 
+SET(DELAYED 
```

Comment:
Any reason to put this up here and not below, where it's actually used?

Reply:
It's just a definition, it doesn't do anything